silkrpc/context_pool.hpp

```diff
@@ -39,6 +40,7 @@ struct Context {
     std::unique_ptr<CompletionRunner> grpc_runner;
     std::unique_ptr<ethdb::Database> database;
     std::unique_ptr<ethbackend::BackEnd> backend;
+    std::unique_ptr<silkrpc::txpool::TransactionPool> tx_pool;
```

Comment:
Use `txpool::TransactionPool` instead of `silkrpc::txpool::TransactionPool`, adding the namespace `silkrpc::` is redundant here